client/app/config/dashboard-grid-options.js

```diff
@@ -13,7 +13,7 @@ const dashboardGridOptions = {
 };
 
 export default function init(ngModule) {
-  ngModule.constant('dashboardGridOptions', dashboardGridOptions);
+  ngModule.constant('dashboardGridOptions', gridOptions);
```

Comment:
Not sure if it should stay injectable - just remove it and import directly where needed (I think only some services need it)

Reply:
Updated.